lizard_damage/models.py

```diff
@@ -275,6 +280,9 @@ class DamageScenario(models.Model):
     scenario_type = models.IntegerField(
         choices=SCENARIO_TYPES, default=0)
 
+    ahn_version = models.CharField(
+        max_length=2, choices=AHN_VERSIONS, default=2
```

Comment:
Huh? Default is het getal 2, maar de enige keuzes zijn de strings `'2'` en `'3'`?